config/settings.yml

```diff
@@ -152,4 +153,4 @@ paperclip:
 #     "User.email": "email"
 #     "User.FirstName": "first_name"
 #     "User.LastName": "last_name"
-
+#  # user_updater_class_name: DartmouthRadar::UserUpdater
```

Comment:
Let's leave the dartmouth specific naming out of this. I think we can have `SamlAuthentication::UserUpdater`.